commons-app/src/main/java/cz/cuni/mff/xrg/odcs/commons/app/conf/AppConfig.java

```diff
@@ -152,4 +175,57 @@ public AppConfig getSubConfiguration(ConfigProperty namespace) {
     public Properties getProperties() {
         return (Properties) prop.clone();
     }
+
+    private static void postprocess(Properties properties) {
```

Comment:
Because it is also called by static method [AppConfig.loadFrom](https://github.com/UnifiedViews/Core/blob/feature/cryptography/commons-app/src/main/java/cz/cuni/mff/xrg/odcs/commons/app/conf/AppConfig.java#L94).